src/ResourceManager/Compute/Commands.Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -290,5 +297,19 @@ public string Location
                 asyncCmdlet.WriteObject(psObject);
             }
         }
+
+        private VirtualMachineScaleSetIdentity GetVmssIdentityFromArgs()
+        {
+            var isUserAssignedEnabled = !string.IsNullOrWhiteSpace(UserAssignedIdentity);
+            return SystemAssignedIdentity || isUserAssignedEnabled
```

Comment:
@praries880 for consistency with other cmdlets, use `SystemAssignedIdentity.IsPresent` to check for the usage of a `SwitchParameter`

Reply:
done